### PR TITLE
Add msdnet recipe

### PR DESCRIPTION
**Pull request by dmpelt**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dmpelt** [comment]
Took a few tries, but I was finally able to get this working on all platforms!

@conda-forge/help-python-c can you review this?

Do you want me to create a new PR first with everything squashed to a single commit?

File: recipes/msdnet/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (xhochy) [inline recipes/msdnet/meta.yaml]: Please remove all the comments as said in the description here.

Comment (xhochy) [reply recipes/msdnet/meta.yaml]: All the comments from the example, not all comments ;)

Comment (dmpelt) [reply recipes/msdnet/meta.yaml]: Done! Sorry about that. I hadn't added any of my own comments, so for now all comments were removed.

**Pull request merged**